src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,181 @@ tags: runtime-traditionalweb;
+    * Review Order
+    * Confirm Order 
 
-## Input Parameters
+    ![](images/wizard-image-20.png)
 
-### Wizard
+1. From the Element Tree, create an Input Parameter by right-clicking on your screen, and from the drop-down, select **Add Input Parameter**. 
+ 
+    The input parameter controls which Wizard step is shown to the user. 
+    
+    ![](images/wizard-image-25.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Orientation  |  Set the orientation. | Orientation Identifier | False | Entities.Orientation.Horizontal |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
+1. On the **Properties** tab, set the Input Parameter properties as follows: 
+    * Name: CurrentStep 
+    * Data Type: Integer
+    * Is Mandatory: No
+    * Default Value: 1
 
-### Wizard Item
+    ![](images/wizard-image-13.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Step  | Set the step. | Step Identifier | True | none |
-| UseTopLabel  |  If true, label is placed above the icon. |  Boolean | False | False |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
-  
-## Layout and Classes
+     By setting the **Default Value** to 1, you ensure that the  Wizard always starts on step 1.
 
-![](images/wizard-image-3.png?width=750)
+1. From the Toolbox, drag two Button widgets onto your screen.
 
-## CSS Selectors
+1. On the **Properties** tab, define the button names and the button behavior (previous and next).
 
-| **Element** |  **CSS Class** |  **Description**  |
-| ---|---|---
-| .wizard |  .wizard.wizard-vertical|  When the wizard orientation is vertical  |
-| .wizard-item |  .wizard-item.active|  Defines the current active step  |
-| .wizard-item |  .wizard-item.past|  Defines the previous step  |
-| .wizard-item |  .wizard-item.next|  Defines the next step  |
+    | **Property** |  **Button 1** | **Button 2**
+    |---|---|---|
+    | Label |  "Previous"| "Next"|
+    | Method |  Navigate| Navigate|
+    | Destination  | Current Screen |  Current Screen|
+    | CurrentStep | CurrentStep - 1 | CurrentStep + 1
 
-## Advanced Use Case
+    ![](images/wizard-image-15.png)    
 
-### Use Wizard with ListRecords
+   You have now set the **On Click** behaviour for each of the buttons to:
 
-1. Drag the Wizard Pattern into the screen.
+    * **navigate** to the **current screen**, and to
+    * update the value of the **CurrentStep** input parameter to the next (CurrentStep + 1) or previous step (CurrentStep - 1) in the Wizard.
 
-1. In the Content placeholder, drag a ListRecords widget.
 
-1. Set the Line Separator parameter of the ListRecords to None.
+1. To create the logic that defines the status of each step (active, past, next), select the first Wizard Item (step 1), and on the **Properties** tab, from the **Step** drop-down, select **Expression Editor**. 
```

Comment:
I'm not sure that "the logic that defines the status of each step (active, past, next)" is really clear.
Perhaps something like
"To create the logic that defines the status of each Wizard Item (Active Step, a Previous Step, or an Upcoming Step)"?